<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UICtags2Api</class>
 <widget class="QDialog" name="UICtags2Api">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Api File Generator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/Ctags2Api.png</normaloff>:/icons/Ctags2Api.png</iconset>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="lInformations">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Api File Generator</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lCtagsBinary">
     <property name="text">
      <string>ctags binary :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="leCtagsBinary">
     <property name="toolTip">
      <string>Locate your ctags binary file</string>
     </property>
     <property name="text">
      <string>ctags-exuberant</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="tbCtagsBinary">
     <property name="icon">
      <iconset>
       <normaloff>:/tools/icons/tools/file.png</normaloff>:/tools/icons/tools/file.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lGenerateFrom">
     <property name="text">
      <string>Generate from :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="cbGenerateFrom">
     <property name="toolTip">
      <string>Generate from a include sources directory or already made ctags file</string>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <item>
      <property name="text">
       <string>Include Directory</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ctags File</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lPoint">
       <property name="text">
        <string> :</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="leLabel">
     <property name="toolTip">
      <string>Locate include sources directory / ctags file</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="tbBrowse">
     <property name="icon">
      <iconset>
       <normaloff>:/tools/icons/tools/icon.png</normaloff>:/tools/icons/tools/icon.png</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lSrcPath">
     <property name="text">
      <string>Src Path :</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="leSrcPath">
     <property name="toolTip">
      <string>Ctags don't stock absolute file path in file, locate the original directory from where you generated this ctags file</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QToolButton" name="tbSrcPathBrowse">
     <property name="icon">
      <iconset>
       <normaloff>:/tools/icons/tools/icon.png</normaloff>:/tools/icons/tools/icon.png</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QGroupBox" name="gbOptions">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="0" colspan="3">
       <widget class="QCheckBox" name="cbRemovePrivate">
        <property name="toolTip">
         <string>Remove private declarations</string>
        </property>
        <property name="text">
         <string>Remove Private</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cbWindowsMode">
        <property name="toolTip">
         <string>Windows mode ( only keep one of the 2 windows api members ending with A or W )</string>
        </property>
        <property name="text">
         <string>Windows Mode, include :</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbLetter">
        <property name="toolTip">
         <string>Choose ending letter you want to keep</string>
        </property>
        <item>
         <property name="text">
          <string>A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>W</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QProgressBar" name="pbLoading">
     <property name="minimumSize">
      <size>
       <width>1</width>
       <height>0</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>369</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="dbbButtons">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../monkey/src/resources/resources.qrc"/>
  <include location="../resources/Ctags2Api.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dbbButtons</sender>
   <signal>accepted()</signal>
   <receiver>UICtags2Api</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dbbButtons</sender>
   <signal>rejected()</signal>
   <receiver>UICtags2Api</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbGenerateFrom</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>lLabel</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbWindowsMode</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbLetter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
